website/src/documentation/sdks/python-type-safety.md

```diff
@@ -64,22 +84,75 @@ To specify type hints as properties of a `DoFn` or `PTransform`, use the decorat
 
 The following code declares an `int` type hint on `FilterEvensDoFn`, using the decorator `@with_input_types()`.
 
+```py
+{% github_sample /apache/beam/blob/master/sdks/python/apache_beam/examples/snippets/snippets_test.py tag:type_hints_do_fn %}
 ```
-{% github_sample /apache/beam/blob/master/sdks/python/apache_beam/examples/snippets/snippets_test.py tag:type_hints_do_fn %}```
 
 Decorators receive an arbitrary number of positional and/or keyword arguments, typically interpreted in the context of the function they're wrapping. Generally the first argument is a type hint for the main input, and additional arguments are type hints for side inputs.
 
-### Defining Generic Types
+### Declaring Type Hints Using Annotations
+
+_New in version 2.21.0._
```

Comment:
Is this also added to the changes.md ?

Reply:
Yes: https://github.com/apache/beam/pull/10717/files#diff-8b1c3fd0d4a6765c16dfd18509182f9d